omod/src/main/webapp/resources/scripts/drugOrders.js

```diff
@@ -199,7 +199,8 @@ angular.module('drugOrders', ['orderService', 'encounterService', 'uicommons.fil
                 var encounterContext = {
                     patient: config.patient,
                     encounterType: config.drugOrderEncounterType,
-                    location: null // TODO
+                    location: null, // TODO
+                    encounterRole: config.encounterRole
```

Comment:
In this object, I just set the values of some properties i get from controller to access it later while constructing json object before submission of drug order. encounterProviders property is set like bellow in order-entry.js file.

encounterProviders: [
     { provider: provider.uuid, encounterRole: encounterContext.encounterRole.uuid }
]